src/tests/rect.c

```diff
@@ -246,6 +246,49 @@ GRAPHENE_TEST_UNIT_BEGIN (rect_interpolate)
 }
 GRAPHENE_TEST_UNIT_END
 
+GRAPHENE_TEST_UNIT_BEGIN (rect_scale)
+{
+  graphene_rect_t unscaled[] = {
+    GRAPHENE_RECT_INIT (  0.f,  0.f, 10.f, 10.f),
+    GRAPHENE_RECT_INIT ( -2.f, -2.f, -2.f, -2.f),
+    GRAPHENE_RECT_INIT ( -4.f,  2.f, 10.f, 10.f),
+    GRAPHENE_RECT_INIT (  1.f,  0.f, 10.f, -1.f),
+  };
+
+  graphene_rect_t scaled[] = {
+    GRAPHENE_RECT_INIT (  0.f,   0.f, 10.f, 10.f),
+    GRAPHENE_RECT_INIT ( -8.f, -20.f,  4.f, 10.f),
+    GRAPHENE_RECT_INIT ( -1.f,   1.f, 2.5f,  5.f),
+    GRAPHENE_RECT_INIT (  0.f,   0.f,  0.f, 10.f),
+  };
+
+  /* (horizontal scale, vertical scale) */
+  float scales[] = {
+    1.f, 1.f,
+    2.f, 5.f,
+    0.25f, 0.5f,
+    0.f, -10.f,
+  };
+
+  int i;
+
+  g_assert_true (G_N_ELEMENTS (unscaled) == G_N_ELEMENTS (scaled));
```

Comment:
You can use `g_assert_cmpint (G_N_ELEMENTS (unscaled), ==, G_N_ELEMENTS (scaled));` instead.

Reply:
Done